src/Data/ArrayBuffer/DataView.purs

```diff
@@ -70,13 +70,13 @@ foreign import byteLength :: DataView -> ByteLength
 
 type Endianness = Boolean
 
-foreign import getterImpl :: forall e r. Fn7 (r -> Maybe r) (Maybe r) String ByteLength Endianness DataView ByteOffset (Eff (arrayBuffer :: ARRAY_BUFFER | e) (Maybe r))
+foreign import getterImpl :: ∀ r. Fn7 (r -> Maybe r) (Maybe r) String ByteLength Endianness DataView ByteOffset (Effect (Maybe r))
```

Comment:
I don't have a strong opinion on `forall`. If these are changed it should be done globally.

Reply:
Updated bower to reflect the merge of [zaquest/purescript-uint/#6](https://github.com/zaquest/purescript-uint/pull/6).